frontends/p4/toP4/toP4.cpp

```diff
@@ -1051,25 +1051,35 @@ bool ToP4::preorder(const IR::IfStatement* s) {
     visit(s->condition);
     builder.append(") ");
     if (!s->ifTrue->is<IR::BlockStatement>()) {
+        builder.append("{");
         builder.increaseIndent();
         builder.newline();
         builder.emitIndent();
     }
     visit(s->ifTrue);
-    if (!s->ifTrue->is<IR::BlockStatement>())
+    if (!s->ifTrue->is<IR::BlockStatement>()) {
+        builder.newline();
         builder.decreaseIndent();
+        builder.emitIndent();
+        builder.append("}");
+    }
     if (s->ifFalse != nullptr) {
         builder.newline();
         builder.emitIndent();
```

Comment:
maybe remove these two, so `else` ends up on the same line as the previous `}`...